scripts/main/sidebar.js

```diff
@@ -328,12 +355,19 @@ sidebar.createStructure.album = function(data) {
 		]
 	};
 
+	videoCount = 0;
+	$.each(data.photos, function () {
+		if (this.type && this.type.indexOf('video') > -1) {
+			videoCount++;
+		}
+	});
 	structure.album = {
 		title : lychee.locale['ALBUM_ALBUM'],
 		type  : sidebar.types.DEFAULT,
 		rows  : [
 			{ title: lychee.locale['ALBUM_CREATED'], kind: 'created',       value: data.sysdate },
-			{ title: lychee.locale['ALBUM_IMAGES'],  kind: 'images',        value: data.photos.length }
+			{ title: lychee.locale['ALBUM_IMAGES'],  kind: 'images',        value: data.photos.length - videoCount },
+			{ title: lychee.locale['ALBUM_VIDEOS'],  kind: 'videos',        value: videoCount }
```

Comment:
only show line 370 if videoCount > 0.

Reply:
Will do...

Reply:
Done. While I was testing it, I noticed that the image count was misbehaving for empty albums so I fixed that as well.